fulfil_client/client.py

```diff
@@ -46,6 +47,8 @@ def wrapper(*args, **kwargs):
                 # logged out. Either way raise this error so the app
                 # can decide how to handle logouts.
                 raise AuthenticationError(loads(rv.text), rv.status_code)
+            elif rv.status_code == 429:
+                raise RateLimitError(loads(rv.text), rv.status_code)
```

Comment:
rv.text will be json?

Reply:
It is not, I realised it later. Can server return that to make it consistent with other response codes?